sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamSuite.scala

```diff
@@ -829,6 +955,18 @@ class StreamSuite extends StreamTest {
       assert(query.exception.isEmpty)
     }
   }
+
+  private def waitForLastExecution(q: StreamExecution): Unit = {
```

Comment:
You can use `eventually` to replace this method.

Reply:
Thanks for letting me know! Addressed.